app/ide-desktop/lib/dashboard/src/authentication/service.ts

```diff
@@ -167,10 +167,11 @@ function loadAmplifyConfig(
     // needs to be registered.
     setDeepLinkHandler(logger, navigate)
   }
+
+  const redirectUrl = process.env.ENSO_CLOUD_REDIRECT ?? window.location.origin
```

Comment:
nit: we might also want to use `APP_BASE_URL`, to cover the rare case when the app is not hosted at `/` (e.g. it's being accessed through a proxy)

Reply:
Maybe we can use the full URL instead, wdyt?

Reply:
that might work, yeah. note that we use path for react-router on cloud.enso.org, but if this behavior does not exist on cloud.enso.org then it's not a problem

(on second thought, it shouldn't be a problem either way right? because if we end up at an invalid path, the react router `Layout`s should take care of that. (and if they don't, then it's a bug that should be fixed anyway)

either way should probably be tested on both electron and browser to make sure nothing breaks

Reply:
>  concat `ENSO_CLOUD_REDIRECT` with `APP_BASE_URL`

yeah this is what i was imagining. not sure whether it's necessary though (see above)

Reply:
Yeah, as far as I remember we redirect all unknown paths to the Dashboard: https://github.com/enso-org/enso/blob/f0411766fa16782a588e1cd50c32f5ba4fccf3f0/app/ide-desktop/lib/dashboard/src/App.tsx#L450

Reply:
Not sure it's inside the scope of the current issue, created a separate: https://github.com/enso-org/cloud-v2/issues/1240